salt/metalk8s/orchestrate/worker_plane.sls

```diff
@@ -0,0 +1,73 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%- set control_plane_ip = control_plane_ips[pillar['bootstrap_id']][0] %}
+{%- else %}
+{%- set control_plane_ip = 'localhost' %}
+{%- endif %}
+
+{% set node = pillar['node_name'] %}
+
+Run bootstrap prechecks:
+  salt.state:
+    - tgt: {{ node }}
+    - saltenv: {{ saltenv }}
+    - sls: metalk8s.bootstrap.precheck
+
+Run bootstrap kubelet:
+  salt.state:
+    - tgt: {{ node }}
+    - saltenv: {{ saltenv }}
+    - sls:
+      - metalk8s.bootstrap.kubelet_install
```

Comment:
minor: always use the same indentation style: either `sls: toto` or `sls: \newline - toto`